fgcm/fgcmDeltaAper.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, fgcmConfig, fgcmPars, fgcmStars):
             self.fgcmLog.warn('No aperture radii set.  Epsilon is unnormalized.')
             self.epsilonNormalized = False
 
-        self.njyZp = 48.6 - 9*2.5
+        self.njyZp = 8.9 + 9*2.5
```

Comment:
I know I'm not technically required to review this PR, but woah, that's a huge drop. What happened here?

Reply:
I got my units confused completely on cgs vs janskys.